src/reducers/breakpoints.js

```diff
@@ -11,34 +11,31 @@
 import * as I from "immutable";
 import { omit } from "lodash";
 import makeRecord from "../utils/makeRecord";
+import { createSelector } from "reselect";
+
 import { isGeneratedId } from "devtools-source-map";
 import { prefs } from "../utils/prefs";
+import transform from "lodash/transform";
 import {
   createPendingBreakpoint,
-  locationMoved,
   makeLocationId,
-  makePendingLocationId,
-  allBreakpointsDisabled
+  makePendingLocationId
 } from "../utils/breakpoint";
 
 import type { Breakpoint, PendingBreakpoint, Location } from "../types";
 import type { Action } from "../actions/types";
 import type { Record } from "../utils/makeRecord";
 
 export type BreakpointMap = I.Map<string, Breakpoint>;
-export type PendingBreakpointMap = I.Map<string, PendingBreakpoint>;
 
 export type BreakpointsState = {
-  breakpoints: BreakpointMap,
-  pendingBreakpoints: PendingBreakpointMap,
-  breakpointsDisabled: false
```

Comment:
it's great to see both `pendingBreakpoints` and `breakpointsDisabled` go away. Both are now simple selectors